webapp/Brocfile.js

```diff
@@ -2,7 +2,11 @@
 
 var EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
-var app = new EmberApp();
+var app = new EmberApp(/*{
+  vendorFiles: {
```

Comment:
Was getting "Uncaught Error: Could not find module handlebars".
From what I understood the change will be needed with Handlebars 2.0

I didn't notice I left it commented. I'll delete it